packages/explorer/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@latticexyz/explorer",
+  "version": "2.1.0",
+  "private": true,
+  "description": "Explorer is a tool for visually exploring and manipulating the state of worlds",
+  "bin": {
+    "explorer": "./explorer-runner.mjs"
```

Comment:
do we need to build this?

Reply:
it doesn't have to be built since it can be used without transpilers but this comment made me realize that perhaps this script should be converted to typescript, as it's done in `store-indexer` package, etc. So changed it to `explorer.ts` and linking the `explorer` command to the transpiled executable i.e. `./dist/explorer.mjs`.